test/IntegrationTests/BuildTests.DistributionStructure_windows.verified.txt

```diff
@@ -261,10 +288,17 @@
   \store\x86\netcoreapp3.1\microsoft.extensions.options.configurationextensions\6.0.0\lib\netstandard2.0\Microsoft.Extensions.Options.ConfigurationExtensions.dll,
   \store\x86\netcoreapp3.1\microsoft.extensions.options\6.0.0\lib\netstandard2.1\Microsoft.Extensions.Options.dll,
   \store\x86\netcoreapp3.1\microsoft.extensions.primitives\6.0.0\lib\netcoreapp3.1\Microsoft.Extensions.Primitives.dll,
-  \store\x86\netcoreapp3.1\mongodb.bson\2.3.0\lib\netstandard1.5\MongoDB.Bson.dll,
-  \store\x86\netcoreapp3.1\mongodb.driver.core.extensions.diagnosticsources\1.1.0\lib\netstandard2.0\MongoDB.Driver.Core.Extensions.DiagnosticSources.dll,
-  \store\x86\netcoreapp3.1\mongodb.driver.core\2.3.0\lib\netstandard1.5\MongoDB.Driver.Core.dll,
-  \store\x86\netcoreapp3.1\system.diagnostics.diagnosticsource\5.0.0\lib\netstandard1.3\System.Diagnostics.DiagnosticSource.dll,
```

Comment:
@rajkumar-rangaraj, side-effect of bump. New version references DiagnosticSource v 6 so it was automatically removed.